### ISSUE TITLE
Develop initial components of React admin and host the demo project in netlify 

### ISSUE THREAD

**Issue by razakpm** [medigy/react-admin-mds-experiments#1]
start to implement the Salesforce Lightning Design System (SLDS) components in the react-admin theme demo. initially use components where we don't have to change the logic of react-admin.

**Comment by razakpm** [comment]
From:Alqama
I started implementing the Salesforce Lightning Design System (SLDS) components in the react-admin theme demo. Currently, I am only using the components where we don't have to change the logic of react-admin.

For the Data Table, I had to implement the component with some logic as it was necessary. I think I just need around 10 more hours or maybe less to complete the task. Please check the deployed site at [Netlify ](https://elated-mclean-9053af.netlify.app/#/login) and let me know if you have any suggestions. Furthermore, I also attached the screenshots of the pages that I converted using SLDS components

**Comment by alqamabinsadiq** [comment]
I can only change those components in the demo which are directly coming from the MATUI and not from `react-admin`, because the components that are coming directly from the `react-admin` requires some logic change. I think the best way to use the react-admin with SLDS would be to change the material-ui inside the react-admin package, but there are some drawbacks in doing that.

**Comment by razakpm** [comment]
Yes agreed, You not need to modify the react-admin package. we will replace the component wherever necessary.

**Comment by alqamabinsadiq** [comment]
If we modify the react-admin package then we won't be able to get the latest updates. Technically, it will be like our own new admin panel implementation and we will have to manage everything on our own.

**Comment by razakpm** [comment]
@alqamabinsadiq We are waiting to get the Typescript file wherever possible. also could you please give a timeline for this work?

**Comment by alqamabinsadiq** [comment]
@razakpm Please check the latest PR.

**Comment by alqamabinsadiq** [comment]
Please check: PR: https://github.com/medigy/react-admin-mds-experiments/pull/2

**Comment by alqamabinsadiq** [comment]
After merging PR: #2, Please let me know how should I proceed. Most of the components you mentioned in the email are coming from `react-admin` hence I can't use SLDS for them.
